pandora-client-web/src/components/header/leaveButton.scss

```diff
@@ -0,0 +1,14 @@
+.LeaveDialog {
+	fieldset {
```

Comment:
As you already show so much info, maybe it might be good to also make it copy-able using `user-select: text;`?